src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final int AMOUNT_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(toFileName);
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(file, true));) {
+            bufferedWriter.write(getReport(fromFileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file " + toFileName, e);
+        }
+    }
+
+    private StringBuilder redFromFile(String fromFileName) {
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/197205133-623cb6ce-e82a-4b21-94ac-937ea3555177.png)
My previous comment not fixed